README.md

```diff
@@ -56,12 +65,12 @@ The following commands install and run the development version of The Lounge:
 ```sh
 git clone https://github.com/thelounge/lounge.git
 cd lounge
-npm install
-NODE_ENV=production npm run build
-npm start
+yarn
```

Comment:
I know it's not necessary, but I would document it as `yarn install` to be a bit more explicit. nbd though